test/test_workspaces.py

```diff
@@ -25,9 +25,9 @@ def test_ensure_workspace_marker(self):
             shutil.rmtree(root_dir)
 
     def test_get_spaces(self):
-        self.assertEqual([], get_spaces([]))
+        root_dir = tempfile.mkdtemp()
         try:
-            root_dir = tempfile.mkdtemp()
+            self.assertEqual([], get_spaces([]))
```

Comment:
Please move this line back outside where it was since it can't raise and doesn't need the `root_dir`.

Reply:
Done.